src/app/views/datasets-new/create/CreateDatasetController.jsx

```diff
@@ -90,11 +90,12 @@ export class CreateDatasetController extends Component {
         recipes.items.forEach(recipe => {
             if (recipe.output_instances.length > 1) {
                 recipe.output_instances.forEach(output => {
-                    allOutputs.push(output);
+                    allOutputs.push(Object.assign({ format: recipe.format }, output));
```

Comment:
nowadays we use spread operator, what do you think simplifying it to : 
```suggestion
                    allOutputs.push({... output,  format: recipe.format  } );
```

Reply:
Yes I did try the spread operator but I couldn't get it to work; I can see where I was going wrong now 😄